app/utils/stripe.js

```diff
@@ -0,0 +1,45 @@
+/* globals Stripe */
+
+import jQuery from 'jquery';
+import Ember from 'ember';
+import config from '../config/environment';
+
+var loadStripe;
+var stripeMock = {};
+
+if(config.env === 'test') {
+  loadStripe = Ember.RSVP.resolve;
+  createStripeToken = Ember.RSVP.resolve;
```

Comment:
The use of test-only code here feels like a code smell but I'm having trouble figuring out a good way to avoid it. It would probably be better to at least have the exported stripeMock have the same API as the Stripe global, i.e.:

```javascript
var stripeMock = {
  setPublishableKey: Ember.K,
  card: {
    createToken: function(){ return Ember.RSVP.resolve(); }
  }
};
``` 

?